cmake/DD4hepConfig.cmake.in

```diff
@@ -26,79 +26,60 @@ set ( DD4hep_ROOT               "${_thisdir}" )
 set ( DD4hep_VERSION            "@DD4hep_VERSION@" )
 set ( DD4hep_SOVERSION          "@DD4hep_SOVERSION@" )
 set ( DD4HEP_USE_GEANT4         "@DD4HEP_USE_GEANT4@" )
+set ( DD4HEP_USE_LCIO           "@DD4HEP_USE_LCIO@" )
 set ( DD4HEP_USE_XERCESC        "@DD4HEP_USE_XERCESC@" )
 set ( DD4hep_BUILD_CXX_STANDARD @CMAKE_CXX_STANDARD@ )
 
```

Comment:
Should we write here which versions were used in the build?

Reply:
Maybe it would be also good to drop `DD4hep_DIR` in favour of `DD4hep_ROOT`, there is no good reason there are two of those

Reply:
There is just one use of DD4hep_ROOT, and many of DD4hep_DIR, so I would leave as is.